### PR TITLE
SAS-17 #create add transaction method and fixed facility related methods

### PR DESCRIPTION
**Pull request by AmitNachimovitz**

### CODE REVIEW

File: src/main/java/com/SAS/team/Team.java [hunk 1]
```diff
@@ -134,18 +144,27 @@ public void setName(String name) {
      * The function return the home stadium of the team
      * @return
      */
-    public Stadium getHomeStadium() {
-        return homeStadium;
+    public Facility getHomeStadium() {
+        for(Facility facility: teamFacilities){
+            if (facility.facilityType == STADIUM){
```
Comment (yaarm) [inline src/main/java/com/SAS/team/Team.java]: I think it should be facility.facilityType == facilityType .STADIUM.

Comment (AmitNachimovitz) [reply src/main/java/com/SAS/team/Team.java]: You're right. I'll fix it.

**Review by yaarm — APPROVED** [review]

**Comment by MrBahur** [comment]
how did you approved the merge when it conflict? please reslove the confflict and only then approve

**Review by MrBahur — CHANGES_REQUESTED** [review]
please fix the conflict code and only then merge.

**Pull request merged**